pages/common/phpcbf.md

```diff
@@ -0,0 +1,28 @@
+# phpcbf
+
+> Fix violations detected by phpcs.
+> More information: <https://github.com/squizlabs/PHP_CodeSniffer>.
+
+- Fix issues in the specified directory (defaults to the PEAR standard):
+
+`phpcbf {{path/to/directory}}`
+
+- Display a list of installed coding standards:
+
+`phpcbf -i`
+
+- Specify a coding standard to validate against:
+
+`phpcbf {{path/to/directory}} --standard {{standard}}`
+
+- Specify file extension(s) to include when sniffing:
```

Comment:
Are they comma-separated? We should clarify that.

Reply:
I created a PR for this here: https://github.com/tldr-pages/tldr/pull/4429